windows/hid.c

```diff
@@ -1313,6 +1315,17 @@ int HID_API_EXPORT HID_API_CALL hid_get_feature_report(hid_device *dev, unsigned
 	return hid_get_report(dev, IOCTL_HID_GET_FEATURE, data, length);
 }
 
+int HID_API_EXPORT HID_API_CALL hid_send_output_report(hid_device* dev, const unsigned char* data, size_t length)
+{
+	BOOL res = HidD_SetOutputReport(dev->device_handle, (PVOID)data, length);
```

Comment:
I'm afraid this is not enought

take a look at `hid_send_feature_report` and try to extract common aproach/implementation, so that the two function would behavio alike

Reply:
I'm not sure why I marked this as resolved - it is clearly not.
See here:
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/hidsdi/nf-hidsdi-hidd_setoutputreport#remarks
and see `hid_send_feature_report` implementation.

Reply:
This is fixed in our branch. I think something happened with the references when I rebased on the latest master branch using the update button from this pull request (probably should have done a merge instead of rebase?).

Reply:
There are inifinite ways one can mess up git history. And there is (almost) always ways to fix it. Give it a try.